tests/integration-tests/tests/storage/test_efs.py

```diff
@@ -106,11 +134,13 @@ def test_multiple_efs(
     new_efs_mount_dirs = ["/shared"]  # OSU benchmark relies on /shared directory
 
     _assert_subnet_az_relations(region, vpc_stack, expected_in_same_az=False)
-    # TODO: change cluster configuration file to test different tls and iam settings to EFS.
+    # change cluster configuration file to test different tls and iam settings to EFS.
     cluster_config = pcluster_config_reader(
         existing_efs_mount_dirs=existing_efs_mount_dirs,
         existing_efs_ids=existing_efs_ids,
         new_efs_mount_dirs=new_efs_mount_dirs,
+        iam_authorizations=iam_authorizations,
+        encryption_in_transits=encryption_in_transits,
```

Comment:
This should only be done is scheduler is not AWS Batch

Reply:
This is not necessary as discussed here https://github.com/aws/aws-parallelcluster/pull/4668#discussion_r1048910822